roles/openshift_hosted_templates/files/v3.6/origin/registry-console.yaml

```diff
@@ -100,9 +100,9 @@ objects:
     redirectURIs:
       - "${COCKPIT_KUBE_URL}"
 parameters:
-  - description: "Container image name"
-    name: IMAGE_NAME
-    value: "cockpit/kubernetes"
+  - description: 'Specify "registry/namespace" prefix for container image; e.g. for "registry.example.com/cockpit/kubernetes:latest", set prefix "registry.example.com/cockpit/"'
```

Comment:
It's possible that someone needs to specify a container name that doesn't include kubernetes.  We shouldn't hard code that above and we should leave this default alone.

Reply:
See the invoking code; nothing ever actually set `IMAGE_NAME`, unless someone manually instantiated the template. Essentially the template default was the hardcoded name.